pkg/cmd/server/start/start_allinone.go

```diff
@@ -132,29 +134,20 @@ func GetAllInOneArgs() (*MasterArgs, *NodeArgs, *ListenArg, *ImageFormatArgs, *K
 	masterArgs.KubeConnectionArgs = kubeConnectionArgs
 	nodeArgs.KubeConnectionArgs = kubeConnectionArgs
 
-	certArgs := NewDefaultCertArgs()
-	masterArgs.CertArgs = certArgs
-	nodeArgs.CertArgs = certArgs
-	kubeConnectionArgs.CertArgs = certArgs
-
-	return masterArgs, nodeArgs, listenArg, imageFormatArgs, kubeConnectionArgs, certArgs
+	return masterArgs, nodeArgs, listenArg, imageFormatArgs, kubeConnectionArgs
 }
 
 func (o AllInOneOptions) Validate(args []string) error {
 	if len(args) != 0 {
 		return errors.New("no arguments are supported for start")
 	}
-	if o.WriteConfigOnly {
-		if len(o.MasterConfigFile) == 0 {
-			return errors.New("--master-config is required if --write-config is true")
-		}
-		if len(o.NodeConfigFile) == 0 {
-			return errors.New("--node-config is required if --write-config is true")
-		}
+
```

Comment:
if ConfigDir.Provided(), ensure len(ConfigDir.Value()) > 0

ensure ConfigDir is not specified along with MasterConfigFile or NodeConfigFile